### ISSUE TITLE
Breaks with custom scalars

### ISSUE THREAD

**Issue by Meemaw** [graphql-python/graphene-federation#13]
Version 3.1.0 breaks with custom scalars.

To reproduce:

```py
class AddressScalar(Scalar):
    base = String

    @staticmethod
    def coerce_address(value: Any):
        ...

    serialize = coerce_address
    parse_value = coerce_address

    @staticmethod
    def parse_literal(ast):
        ...


class Query(ObjectType):
    test = String(x=AddressScalar())


schema = build_schema(query=Query)
```

**Comment by patrick91** [comment]
@Meemaw can you post the error? 😊

**Comment by arunsureshkumar** [comment]
> @Meemaw can you post the error? 😊

@patrick91 - This issue will be resolved in the https://github.com/graphql-python/graphene-federation/pull/14

**Comment by patrick91** [comment]
@arunsureshkumar do you have time to check at this error too? :)

**Comment by arunsureshkumar** [comment]
> @arunsureshkumar do you have time to check at this error too? :)

Sure!

**Comment by arunsureshkumar** [comment]
https://github.com/graphql-python/graphene-federation/pull/15 - @patrick91 Please review.

**Comment by patrick91** [comment]
@Meemaw should be fixed in v3.1.2 :)

**Comment by patrick91** [comment]
sorry, it should be 3.1.3, I forgot to update the version in setup.py 🤦
